packages/turbo-repository/__tests__/find.test.ts

```diff
@@ -58,11 +58,6 @@ describe("Workspace", () => {
           { name: "ui", relativePath: "packages/ui" },
         ],
       },
-      {
```

Comment:
I removed this test because without access to `globalDependencies`, you can't really know if a global change can be ignored. For all we know, a user could have `README.md` in their `globalDependencies` array.

Reply:
`globalDependencies` is not a known concept here. `@turbo/repository` does not assume the presence of a `turbo.json` file.

Can you put the test back and change the `expected` value instead? description can be something like `global change that probably should be irrelevant`. I want to be clear that this is documented/intentional behavior in the tests

Reply:
Done!